source/common/runtime/runtime_impl.h

```diff
@@ -46,13 +47,14 @@ class RandomGeneratorImpl : public RandomGenerator {
  */
 // clang-format off
 #define ALL_RUNTIME_STATS(COUNTER, GAUGE)                                                          \
+  GAUGE  (admin_overrides_active)                                                                  \
+  COUNTER(deprecated_feature_use)                                                                  \
   COUNTER(load_error)                                                                              \
-  COUNTER(override_dir_not_exists)                                                                 \
-  COUNTER(override_dir_exists)                                                                     \
   COUNTER(load_success)                                                                            \
-  COUNTER(deprecated_feature_use)                                                                \
   GAUGE  (num_keys)                                                                                \
-  GAUGE  (admin_overrides_active)
+  GAUGE  (num_layers)                                                                              \
```

Comment:
We probably shouldn't import this gauge into the new process during hot restart? cc @fredlas for why it would be great to encode this as part of the gauge definition. :)

Reply:
Is there a pattern to follow on how to handle this?

Reply:
See https://github.com/envoyproxy/envoy/blob/master/source/common/stats/stat_merger.cc#L17. This is not great and we need to fix this to avoid further problems like this. IIRC there is an open issue. Can you sync up with @fredlas just to make sure this is going to wind up in a near-term future sprint?

Reply:
Yep, fixed, will chat with @fredlas offline.

Reply:
See https://github.com/envoyproxy/envoy/pull/7083 -- that moves the source of truth for whether a gauge is accumulated across  hot-restart to the declaration (e.g. GAUGE macro or Scope::gauge() call).